src/rewrite.rs

```diff
@@ -44,7 +81,6 @@ pub(crate) struct RewriteContext<'a> {
     pub(crate) skip_context: SkipContext,
     pub(crate) skipped_range: Rc<RefCell<Vec<(usize, usize)>>>,
 }
-
```

Comment:
Let's revert this whitespace change. It'll be better to keep a space to separate `RewriteContext` and `InsideMacroGuard`.